size.mjs

```diff
@@ -5,21 +5,21 @@ export default function size (query='') {
 .sb-auto${query}{block-size:auto}
 .sb-100${query}{block-size:100%}
 .sb-min-0${query}{min-block-size:0}
-.sb-min-none${query}{min-block-size:none}
+.sb-min-none${query}{min-block-size:unset}
```

Comment:
I made that change because `none` is not a supported value for `min-block-size`.
I ran the output through a CSS validator which threw the error then I double checked against MDN docs.
Is there a better valid value to use?

Reply:
Sonuvagun! Good spot! The initial value looks to be `0`, so actually it looks like this class and the inline equivalent aren't needed at all, since we have `sb-min-0` and `si-min-0`.

Reply:
More removals!
I will take them out then.